community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseBuilder.java

```diff
@@ -118,6 +119,16 @@ public GraphDatabaseBuilder setConfig( Map<String, String> config )
     }
 
     /**
+     * Retrieve the configured configuration values.
+     *
+     * @return the configured configuration values.
+     */
+    public Config config()
+    {
+        return new Config( config );
+    }
```

Comment:
and looking into how it's used later looks like i was completely wrong so please ignore my comment